awx/main/access.py

```diff
@@ -1604,11 +1605,15 @@ def get_value(Class, field):
         project = get_value(Project, 'project')
         # If the user has admin access to the project (as an org admin), should
         # be able to proceed without additional checks.
-        if project:
-            return self.user in project.use_role
-        else:
+        if not project:
+            return False
+
+        if self.user not in project.use_role:
+            self.messages['job_template'] = [_('You do not have use permission on Project')]
```

Comment:
The field that has the error is "project" not "job_template". The user already knows they are editing a job template. So I would change that, just the _key_ here.